task_reactredux/task1/src/Constants.js

```diff
@@ -0,0 +1,8 @@
+const commandsEnum = {
+    start: "start",
+    add: "add",
+    del: "delete",
+    def: "default"
+}
```

Comment:
First of all, you missed a semicolon. To create a 'real' const value in js, you should use `Object.freeze()`